frontend/components/mobile/MobileSearchInput.tsx

```
@@ -133,13 +133,14 @@ export default function MobileSearchInput(props: MobileSearchInputProps) {
             id="search_input"
             className="w-56 placeholder:text-2xl text-2xl border-b-2 border-gray-400 outline-none rounded-2xl"
             placeholder={`Search ${searchTypeSelected === SearchType.EVENT ? "Events" : "Users"}`}
+            aria-label="Search for sports events"
             value={searchParameter}
             onChange={(event) => {
               setSearchParameter(event.target.value);
             }}
             onKeyDown={handleKeyPress}
           />
-          <button onClick={handleSearch}>
+          <button onClick={handleSearch} aria-label="Search events">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)